profiles/base/package.use.mask

```diff
@@ -7,6 +7,23 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Michał Górny <mgorny@gentoo.org> (2019-12-02)
+# Python 2.7-compatible packages optionally needing dev-python/ipython.
+# Note that all but [doc] flags here are optional runtime deps,
+# so no functionality is lost.
+app-misc/gourmet ipython
+dev-python/jira ipython jirashell
+dev-python/matplotlib doc
```

Comment:
Hi, I don't understand why masking is needed if python2_7 are moved from the dependents of ipython.  Could you please enlighten me?

Reply:
I've presumed that of the two alternatives, temporarily disabling doc building would be less of a problem than killing Python 2 support entirely. Not to mention the latter would imply having to drop it from a whole new bunch of revdeps…

Reply:
OK, I understand now.  A further question taking matplotlib as example: How do you confirm only `dev-python/matplotlib[doc]` depends on` dev-python/ipython[python_targets_python2_7]` but not `dev-python/matplotlib` itself?  Are there some tools to aid this judgement?

Reply:
I presume that whoever bumped it last time had verified the dependencies. Python world is not consistent enough to make it possible for good tooling to be created. It's handiwork if you want to check it.